README.md

```diff
@@ -1,25 +1,25 @@
 # OpenZeppelin Starter Kit
 
-An OpenZeppelin Starter Kit containing React, OpenZeppelin SDK, OpenZeppelin Contracts, Truffle and Infura.
+An OpenZeppelin Starter Kit containing React, OpenZeppelin CLI, OpenZeppelin Contracts, Truffle and Infura.
 
 This kit comes with everything you need to start using upgradeable Smart
 contracts inside your applications. It also includes all the configuration
 required to deploy to different networks.
 
 ## Requirements
 
-Install OpenZeppelin SDK, Ganache, and Truffle
+Install Ganache, and Truffle
 
 ```
-npm install -g truffle@5.0.41 ganache-cli@6.7.0 @openzeppelin/cli@2.5.3
+npm install -g truffle@5.0.41 ganache-cli@6.7.0
```

Comment:
Do we still need to install truffle?

Reply:
The project uses `truffle-config.js` for Network setup and has Truffle tests, both need upgrading to OpenZeppelin CLI and OpenZeppelin Test Environment

Reply:
I _think_ we don't need truffle for the config: the CLI will be able to read network info from `truffle-config.js` (though using our own `network.js` is preferred: we should migrate eventually to that format). The tests should also be easy to migrate from a quick look, I'd do it if we're still pushing for Starter Kit usage.